authors/apps/articles/models.py

```diff
@@ -28,3 +31,18 @@ class FavoriteArticle(models.Model):
 
     def __str__(self):
         return str(self.pk)
+
+
+class ReadingStats(models.Model):
+    """
+    Model for Reading statistics
+    """
+
+    article = models.ForeignKey(Article, on_delete=models.CASCADE)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+
+    class Meta:
+        unique_together = ('article', 'user')
```

Comment:
I see you have a unique together, which prohibits adding user and an article more than once. However I think a user should be able to read an article more than once, and each time the statistics should be captured.

Reply:
@bison, I have removed the unique together validation and now a user can add to the view stats of the article each time they visit the article.